crates/sui-indexer/src/indexer_reader.rs

```diff
@@ -392,6 +392,26 @@ impl IndexerReader {
         Ok(system_state)
     }
 
+    pub fn get_epoch_sui_system_state(
```

Comment:
Can we add a doc comment here? Also, does this actually fetches the live system state (obj 0x5), or just a historical system state when no epoch is passed? 
```suggestion
    /// Retrieve the system state data for the given epoch. If no epoch is given,
    /// it will retrieve the last known epoch's data and return the system state.
    pub fn get_epoch_sui_system_state(
```